<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
  PortableQtApp template
  Copyright (C) 2018-2020 Pedro Lopez-Cabanillas &lt;plcl@users.sf.net&gt;

  This program is free software: you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation, either version 3 of the License, or
  (at your option) any later version.

  This program is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  GNU General Public License for more details.

  You should have received a copy of the GNU General Public License
  along with this program.  If not, see &lt;http://www.gnu.org/licenses/.&gt;
 </comment>
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="1">
      <widget class="QComboBox" name="comboOption">
       <property name="currentText">
        <string/>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Item 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Item 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Item 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Item 4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkOption"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="textOption"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="numberOption"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCheck">
       <property name="text">
        <string>Check Box Option:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCombo">
       <property name="text">
        <string>ComboBox Option:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelText">
       <property name="text">
        <string>TextBox Option:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelNumber">
       <property name="text">
        <string>Numeric Option:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
